### ISSUE TITLE
Black square in Gamemode

### ISSUE THREAD

**Issue by carpaBR** [holoiso-eol/holoiso#411]
i5-8400 with a GTX1060

After install, gamemode was a bit glitchy. Tryng the system, I activeted the system overlay and a black square appered in the superior left corner. Disabled system overlay and the black square continued. In desktop mode everything works fine. Game mode has more than half  the screen blocked. Anything to do?

**Comment by ziqul** [comment]
Same issue.
RTX 2080
AMD Ryzen 7 2700

**Comment by carpaBR** [comment]
i rebuild the mangohub from their github page. Got it fixed, but for nvidia, gamemode is problematic

**Comment by ziqul** [comment]
Can you give a bit more info, please?

I have cloned https://github.com/flightlessmango/MangoHud, built and installed it. Binaries now live in `/usr/lib/mangohud`.
But I still end up with black box when I am launching MangoHud.

**Comment by carpaBR** [comment]
there is not much do add, gamemode had a black square in the upper left corner. I followed the mangohud instructions, had some problems, then at each problem compiling the app, I googled and install the missing lib using pacman. When it finished with no errors, gamemode had no more black square. But still no working, launching the game from gamemode is super glitchy, it keeps restarting the gamemode interface. Desktop mode works fine. I waiting for new realeses to see if works better for Nvidias. With AMD cards, there is no problem.
